federation/executor.go

```diff
@@ -25,8 +26,11 @@ type ExecutorClient interface {
 // subquery on any of the federated servers.
 // The planner allows it to coordinate the subqueries being sent to the federated servers
 type Executor struct {
-	Executors map[string]ExecutorClient
-	planner   *Planner
+	Executors    map[string]ExecutorClient
+	planner      *Planner
```

Comment:
Rename ticker to `schemaSyncerTicker`, or group these three fields into another struct `syncer? `

Reply:
Or put the ticker inside the syncer?